src/main/java/pl/betoncraft/betonquest/config/Config.java

```diff
@@ -370,17 +370,47 @@ public static void sendMessage(String playerID, String messageName,
      *          ID of the message
      * @param variables
      *          array of variables which will be inserted into the string
-     * @param sound
+     * @param soundName
      *          name of the sound to play to the player
      */
     public static void sendMessage(String playerID, String messageName,
             String[] variables, String soundName) {
+        sendMessage(playerID, messageName, variables, soundName, null, null);
+    }
+    
+    /**
+     * Sends a message to player in his chosen language or default
+     * or English (if previous not found). It will replace all {x}
+     * sequences with the variables and play the sound. It will also
+     * add a prefix to the message.
+     * 
+     * @param playerID
+     *          ID of the player
+     * @param messageName
+     *          ID of the message
+     * @param variables
+     *          array of variables which will be inserted into the message
+     * @param soundName
+     *          name of the sound to play to the player
+     * @param prefixName
+     *          ID of the prefix
+     * @param prefixVariables
+     *          array of variables which will be inserted into the prefix
+     */
+    public static void sendMessage(String playerID, String messageName,
+            String[] variables, String soundName, String prefixName, String[] prefixVariables) {
         Player player = PlayerConverter.getPlayer(playerID);
         DatabaseHandler dbHandler = BetonQuest.getInstance().getDBHandler(playerID);
         if (player == null || dbHandler == null) return;
         String language = dbHandler.getLanguage();
         String message = getMessage(language, messageName, variables);
         if (message.length() == 0) return;
+        if (prefixName != null) {
+            String prefix = getMessage(language, prefixName, prefixVariables);
+            if (prefix.length() > 0) {
+                message = prefix + " " + message;
```

Comment:
This way the space it's not removable. The user should be able to choose if he want's it or not by manually adding it in the prefix.